test/src/main/java/org/apache/accumulo/test/functional/KerberosIT.java

```diff
@@ -616,13 +617,13 @@ public void testDelegationTokenWithReducedLifetime() throws Throwable {
             assertEquals(rootUser.getPrincipal(), client.whoami());
 
             return client.securityOperations().getDelegationToken(
-                new DelegationTokenConfig().setTokenLifetime(5, TimeUnit.MINUTES));
+                new DelegationTokenConfig().setTokenLifetime(5, MINUTES));
           }
         });
 
     AuthenticationTokenIdentifier identifier = ((DelegationTokenImpl) dt).getIdentifier();
     assertTrue("Expected identifier to expire in no more than 5 minutes: " + identifier,
-        identifier.getExpirationDate() - identifier.getIssueDate() <= (5 * 60_000));
+        identifier.getExpirationDate() - identifier.getIssueDate() <= (MINUTES.toMillis(5)));
```

Comment:
After the conversion, this has some extra parens that could be removed:
```suggestion
        identifier.getExpirationDate() - identifier.getIssueDate() <= MINUTES.toMillis(5));
```